File: BlogAuthService/AuthService.Core/Emails/EmailExtensionOptions.cs
namespace AuthService.Core.Emails
{
    /// <summary>
    /// Email extension options.
    /// </summary>
    public class EmailExtensionOptions
    {
        /// <summary>
        /// Gets or sets the base URL.
        /// </summary>
        /// <value>
        /// The base URL.
        /// </value>
        public string BaseUrl { get; set; }

        /// <summary>
        /// Gets or sets from.
        /// </summary>
        /// <value>
        /// From.
        /// </value>
        public string From { get; set; }
    }
}

File: Shared/Blog.FSharp.Core/Emails/EmailExtensionOptions.fs
namespace Blog.FSharp.Core.Emails

namespace Blog.Core.Options

/// <summary>
/// Email Extension Options.
/// </summary>
type EmailExtensionOptions() =

    /// <summary>
    /// Gets or sets baseUrl.
    /// </summary>
    member val BaseUrl : string = "" with get, set

    /// <summary>
    /// Gets or sets from.
    /// </summary>
    member val From : string = "" with get, set